common/lib/dependabot/pull_request_updater/azure.rb

```diff
@@ -55,9 +55,16 @@ def update_source_branch
         # 1) Push the file changes to a newly created temporary branch (from base commit)
         new_commit = create_temp_branch
         # 2) Update PR source branch to point to the temp branch head commit.
-        update_branch(source_branch_name, old_source_branch_commit, new_commit)
+        response = update_branch(source_branch_name, old_source_branch_commit, new_commit)
         # 3) Delete temp branch
         update_branch(temp_branch_name, new_commit, OBJECT_ID_FOR_BRANCH_DELETE)
+
+        unless response.fetch(
+          "success", false
+        )
```

Comment:
Nitty, but this is easier to read on a single line IMO

```suggestion
        unless response.fetch("success", false)
```